src/components/app-header.js

```diff
@@ -469,14 +469,7 @@ class AppHeader extends React.PureComponent {
                       styleName='inline.margin-right-one'
                     >
                       <Dropdown.Menu>
-                        <Dropdown.Item>
-                          <Message
-                            negative
-                            icon='hourglass half'
-                            header='Work in progress'
-                            content='Notifications are not availaible at this stage.'
-                          />
-                        </Dropdown.Item>
+                        <NotificationListView class="notifications-header-list"/>
```

Comment:
- [ ]  In react, we use `className` instead of class and in our case we use styleName. Are you sure it is showing the intended behaviour, in addition, if it is for applying some style as we have imported multiple style sheets it must be in the form of `<object>.notifications-header-list`

- [ ]  We use single inverted commas not double ones